examples/cryptopp/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-cryptopp)
+
+hunter_add_package(cryptopp)
+find_package(cryptopp CONFIG REQUIRED)
+
+
+add_executable(test main.cpp)
+
+
+target_link_libraries(test PRIVATE cryptopp-static)
```

Comment:
So it does install static and shared variants? This will work fine and acceptable but in general it should be just `cryptopp`, see http://cgold.readthedocs.io/en/latest/tutorials/libraries/static-shared.html